src/components/app/details/main.tsx

```diff
@@ -161,11 +166,15 @@ export function AppHeader({ appName }: { appName: string }) {
                         activeClassName="active"
                         to={`${match.url}/${URLS.APP_DETAILS}`}
                         className="tab-list__tab-link"
+                        ref={appDetailsRef}
                         onClick={(event) => {
                             ReactGA.event({
                                 category: 'App',
                                 action: 'App Details Clicked',
                             })
+                            const linkDisabled = appDetailsRef.current?.classList.contains('active')
```

Comment:
Please create 1 common method instead of duplicating the same code

Reply:
made changes as suggested